doc/contributor/howto-guide-setup-development-workstation.md

```diff
@@ -61,6 +61,11 @@ sudo update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang
 sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-10 100
 ```
 
+Note: newer versions of Ubuntu might require 
+```console
+sudo apt install -y clang libc++-dev libc++abi-dev cmake ninja-build
```

Comment:
We are about to drop Ubuntu 18.  Maybe just update the instructions to Debian / Ubuntu 22?  Or just remove the obsolete instructions!

Reply:
Is clang 1:1 with clang-10, clang-tidy, and clang-formatter? I'll remove the old instructions if it is

Reply:
I /think/ (emphasis on "I might be wrong") `clang-10` installs an specific version of Clang, while `clang` installs the default version.  The same applies to the other Clang-related packages.  With older versions of Ubuntu we probably wanted to name an specific version because the default was "too old".  With something like Ubuntu 22 (or Debian 11) we are happy with the default.

I am not even sure I even landed in the ballpark of an answer to your question.

Reply:
That explanation makes sense to me. I'll remove the old instructions.

Reply:
Discussed outside of PR. We will be keeping both since other users might have varying Ubuntu versions.